cmd/ftl/cmd_schema_diff.go

```diff
@@ -57,6 +83,60 @@ func (d *schemaDiffCmd) Run(ctx context.Context, currentURL *url.URL) error {
 	return nil
 }
 
+func localSchema(moduleDirs projectconfig.Config) (*schema.Schema, error) {
+	pb := &schemapb.Schema{}
+	found := false
+	tried := ""
+	moduleFound := false
+	moduleTried := ""
+	for _, modulePaths := range moduleDirs.AbsModuleDirs() {
+		entries, err := os.ReadDir(modulePaths)
+		if err != nil {
+			return nil, fmt.Errorf("failed to list directory %w", err)
+		}
+		for _, module := range entries {
+			if !module.IsDir() {
+				continue
+			}
+			path := filepath.Join(modulePaths, module.Name(), ".ftl", "schema.pb")
+			mod, err := loadProtoSchema(path)
+			if err != nil {
+				moduleTried += fmt.Sprintf("failed to read schema file %s; did you run ftl build?", path)
+			} else {
+				moduleFound = true
+				found = true
+				pb.Modules = append(pb.Modules, mod)
+			}
+		}
+	}
+	if !moduleFound {
+		tried += moduleTried
+	}
+
+	if !found {
+		return nil, errors.New(tried)
+	}
+
+	s, err := schema.FromProto(pb)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse local schema: %w", err)
+	}
+	return s, nil
+}
+
+func loadProtoSchema(file string) (*schemapb.Module, error) {
```

Comment:
Use `schema.ModuleFromProtoFile()` instead.

Reply:
That actually returns a different Module, it returns a module from `backend/schema` when we are expecting the one from `backend/protos/xyz/block/ftl/v1/schema`.

Reply:
Actually never mind, I needed to remove some other conversion code